sample.css

```
@@ -0,0 +1,40 @@
+	margin: 0.5rem; /* Extra space before colon */
+	line-height: 0.9; /* Missing leading zero before decimal value (.9 instead of 0.9) */
+}
+
+/* 4. Use of !important (often discouraged) */
+.alert {
+	color: red !important; /* Use of !important */
+}
+
+/* 5. Property with a likely misspelling or non-standard property (stylelint may flag unknown properties) */
+.footer {
+	colr: blue; /* 'colr' is not a valid CSS property */
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)